### ISSUE TITLE
Solstice on AutoPkg 2 error

### ISSUE THREAD

**Issue by marckerr** [autopkg/MLBZ521-recipes#10]
After updating to AutoPKG 2.0.2 I'm getting an error on the Solstice .pkg build.  Here's the run.

```
Processing Test-Solstice.pkg.recipe...
URLDownloader
URLDownloader: Storing new Last-Modified header: Tue, 28 Jan 2020 20:33:09 GMT
URLDownloader: Downloaded /Users/AutoPkgUser/Documents/AutoPkgr/Cache/local.pkg.Test-Solstice/downloads/SolsticeClientMacWeb.zip
Unarchiver
Unarchiver: Guessed archive format 'zip' from filename SolsticeClientMacWeb.zip
Unarchiver: Unarchived /Users/AutoPkgUser/Documents/AutoPkgr/Cache/local.pkg.Test-Solstice/downloads/SolsticeClientMacWeb.zip to /Users/AutoPkgUser/Documents/AutoPkgr/Cache/local.pkg.Test-Solstice/unpack
FileFinder
FileFinder: Found file match: '/Users/AutoPkgUser/Documents/AutoPkgr/Cache/local.pkg.Test-Solstice/unpack/SolsticeClientMacWeb.app' from globbed '/Users/AutoPkgUser/Documents/AutoPkgr/Cache/local.pkg.Test-Solstice/unpack/*.app'
CodeSignatureVerifier
CodeSignatureVerifier: Verifying code signature...
CodeSignatureVerifier: Deep verification enabled...
CodeSignatureVerifier: Strict verification not defined. Using codesign defaults...
CodeSignatureVerifier: /Users/AutoPkgUser/Documents/AutoPkgr/Cache/local.pkg.Test-Solstice/unpack/SolsticeClientMacWeb.app: valid on disk
CodeSignatureVerifier: /Users/AutoPkgUser/Documents/AutoPkgr/Cache/local.pkg.Test-Solstice/unpack/SolsticeClientMacWeb.app: satisfies its Designated Requirement
CodeSignatureVerifier: /Users/AutoPkgUser/Documents/AutoPkgr/Cache/local.pkg.Test-Solstice/unpack/SolsticeClientMacWeb.app: explicit requirement satisfied
CodeSignatureVerifier: Signature is valid
EndOfCheckPhase
PkgRootCreator
PkgRootCreator: Created /Users/AutoPkgUser/Documents/AutoPkgr/Cache/local.pkg.Test-Solstice/pkgroot/Applications
PkgRootCreator
PkgRootCreator: Created /Users/AutoPkgUser/Documents/AutoPkgr/Cache/local.pkg.Test-Solstice/scripts
SolsticeProcessor
Traceback (most recent call last):
  File "/Users/AutoPkgUser/Documents/AutoPkgr/Repos/com.github.autopkg.MLBZ521-recipes/Solstice/SolsticeProcessor.py", line 106, in main
    plist_contents = plist_Reader(plist)
  File "/Library/AutoPkg/Python3/Python.framework/Versions/Current/lib/python3.7/plistlib.py", line 936, in load
    header = fp.read(32)
AttributeError: 'str' object has no attribute 'read'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Library/AutoPkg/autopkglib/__init__.py", line 673, in process
    self.env = processor.process()
  File "/Library/AutoPkg/autopkglib/__init__.py", line 480, in process
    self.main()
  File "/Users/AutoPkgUser/Documents/AutoPkgr/Repos/com.github.autopkg.MLBZ521-recipes/Solstice/SolsticeProcessor.py", line 108, in main
    raise ProcessorError('Unable to locate the specified plist file.')
autopkglib.ProcessorError: Unable to locate the specified plist file.
  File "/Library/AutoPkg/autopkglib/__init__.py", line 673, in process
    self.env = processor.process()
Unable to locate the specified plist file.
Failed.
Receipt written to /Users/AutoPkgUser/Documents/AutoPkgr/Cache/local.pkg.Test-Solstice/receipts/Test-Solstice.pkg-receipt-20200219-111909.plist

The following recipes failed:
    Test-Solstice.pkg.recipe
        Error in local.pkg.Test-Solstice: Processor: SolsticeProcessor: Error: Unable to locate the specified plist file.

The following new items were downloaded:
    Download Path                                                                                          
    -------------                                                                                          
    /Users/AutoPkgUser/Documents/AutoPkgr/Cache/local.pkg.Test-Solstice/downloads/SolsticeClientMacWeb.zip  
```

**Comment by MLBZ521** [comment]
Yeah, I noticed an issue last night on this and a couple other recipes.  Working them today hopefully.

**Comment by MLBZ521** [comment]
@marckerr Sorry for the delay on this.  I'm getting caught up on a lot of things.

This should be resolved for Python3 in 6d5f622.
